.travis.yml

```diff
@@ -38,6 +43,12 @@ env:
     - COVERAGE_REPORTS_TOTAL=4
     - FOREM_OWNER_SECRET="secret" # test secret so e2e tests can run properly.
 before_install:
+  - >-
+    sudo sed -i
+    -e '/local.*peer/s/postgres/all/'
+    -e 's/peer\|md5/trust/g'
+    /etc/postgresql/12/main/pg_hba.conf
+  - sudo service postgresql@12-main restart
```

Comment:
Change permission of withing `pg_hba.conf` so we can continue to access PG without a password